app/registries/foreman/plugin.rb

```diff
@@ -155,6 +156,7 @@ def initialize(id)
       @renderer_variable_loaders = []
       @ping_extension = nil
       @status_extension = nil
+      @global_registration_vars = []
```

Comment:
Why is this an array while the global one is a hash?

Reply:
Here we keep list of allowed params / vars. The other one assigns values to variables.

Reply:
Then I'd reflect that in the variable name
```suggestion
      @global_registration_allowed_vars = []
```

My interpretation of reading the code was that a plugins would assign variables directly this was, rather than an indirection.

Reply:
Oh I see. I'll change the name then.